Challenge two week one.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>CREATION</title>
+<style>
+body{
+font-type : System;
+font-size: 14.5px;
+background-color: white;
+}
+
+#topbar{
+position: fixed;
+right:0;
+min-width: 250px;
+height: 50px;
+padding: 10px;
+background-color: transparent;
+border-color: grey;
+border-width: 2px;
+}
+
+#topbar li{
+color: black;
+}
+#google{
+left: 100px;
+right: 100px;
+top: 500px;
+font: bold;
+}
+#bottombar1{
+border: solid;
+height: 45px;
+border-width: 100%;
+border-color: gray;
+background-color: lightgray;
+border-bottom-width: 1px;
+border-left-width:1px;
+border-right-width: 1px;
+border-top-width: 1px;

```

Comment:
I can see that you are setting all sides with the same value. You don't have to set them individually if they have the same value, `border-width: 1px;` sets all sides.